engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -30,20 +30,22 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
         }
 
         function success(response) {
-            $scope.successMessages = response.displayMessages.success;
-            $scope.errorMessages = response.displayMessages.error;
+            angular.forEach(response.task.input.target_ids, function (row) {
+                $scope.repositoriesTable.removeRow(row);
+            });
+            $scope.successMessages.push("Initiated removal of selected repositories, see task " + response.task.id + " for details.");
```

Comment:
What about linking to this task directly in the success message?  We do something similar [here](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata.html#L10-10).